### ISSUE TITLE
No 'run' instructions in README and no CONTRIBUTING guide

### ISSUE THREAD

**Issue by bvhme** [AjuntamentdeBarcelona/floss-guide-bcn-en#2]
In order to make it easier for others to contribute to this project (also following this guide's `M_4F5`) I think this repository would greatly benefit from a short introduction on how to get the guide to build on your own computer, edit the files (link to introduction to ASCIIDoc) and a guide that tells you how to do a Pull Request and what happens with that.

I think the following are relevant as they also deal with 'content' more than 'code':

[Example of a `README` from from amsterdam.github.io](https://github.com/Amsterdam/amsterdam.github.io/blob/master/README.md)
[Example of a `CONTRIBUTING` file from amsterdam.github.io](https://github.com/Amsterdam/amsterdam.github.io/blob/master/CONTRIBUTING.md)

**Comment by bvhme** [comment]
I would be glad to propose a `CONTRIBUTING` (although you might want to do this yourself). A `README` however I feel not ready for yet.

**Comment by gmarpons** [comment]
You're right @bvhme, we should have README and CONTRIBUTING files for all our Guides, including this one, and for the site repo.

Your contributions are welcome.

Me and @imingueza are on vacation until September 3rd, so I cannot guarantee contributions will be reviewed in the next weeks, but they are not going to be ignored.

Thanks for your interest.
